README.md

```diff
@@ -18,6 +18,33 @@ This action is used by the Dependabot [version][docs-version-updates] and [secur
 To manually upgrade `dependabot-action` on your [GitHub Enterprise Server (GHES)](https://github.com/enterprise), follow [these instructions](https://docs.github.com/en/enterprise-server/admin/managing-github-actions-for-your-enterprise/managing-access-to-actions-from-githubcom/manually-syncing-actions-from-githubcom).
 **Warning:** The current release of `dependabot-action` only guarantees backwards compatibility with the [currently supported GHES versions](https://docs.github.com/en/enterprise-server/admin/all-releases). Once a GHES version is deprecated, future versions of `dependabot-action` may introduce incompatible changes.
 
+##  Steps to Update Your PR and Deploy Changes
+1. Make changes to the PR
+2. Run the following command in your terminal on the same branch used to create the PR whenever you make changes to the PR
+```bash
+npm run lint-check
+npm run format-check -- --write
+npm run test
+```
+
+```
+nvm install;nvm use;npm clean-install;npm ci;npm run package
```

Comment:
Are all of these needed? I thought in the general case we can assume someone is starting with a clean env and run `npm ci`... but maybe I'm wrong and the rest of these are needed?

Reply:
In the codespace, this step is needed.